website/docs/jellyfish/api/spv.md

```diff
@@ -50,3 +50,22 @@ interface ReceivedByAddressInfo {
   txids: string[]
 }
 ```
+
+## sendToAddress
+
+Send a Bitcoin amount to a given address.
+
+```ts title="client.spv.sendToAddress()"
+interface spv {
+  sendToAddress (address: string, amount: BigNumber, options: SendToAddressOptions = { feerate: 10000 }): Promise<SendMessageResult>
```

Comment:
Shouldn't this be 
sendToAddress (address: string, amount: BigNumber, options = { feeRate: new BigNumber('10000') }): Promise<SendMessageResult>

Which is the latest interface you had just changed in your rpc file?